Src/BootCamp.Chapter/MsFactory.cs

```diff
@@ -9,7 +9,17 @@ public class MsFactory
     {
         public DesktopComputer Assemble()
         {
-            return new DesktopComputer();
+            DesktopComputer desktopComputer = new DesktopComputer(new Body(), new Ram(), new Cpu(), new Gpu(), new HardDisk(), new Motherboard());
+
+            desktopComputer.GetBody();
```

Comment:
Instead of ``Get``, use ``Install``. We are supposed to install different parts into a computer.
And each factory installs their own brand parts.